HelpSource/Classes/Array.schelp

```diff
@@ -69,25 +69,35 @@ copymethod:: SequenceableCollection *exprand
 
 copymethod:: SequenceableCollection *fib
 
-
-
-
-
 InstanceMethods::
 
 copymethod:: ArrayedCollection -at
 
-copymethod:: ArrayedCollection -put
+method:: put
+Put item at index, replacing what is there.
+code::
+x=[1, 2,3 , 4, 5]
+x.put(rrand(0, 4), 8)
```

Comment:
I'd remove the ```rrand``` as it makes the example more complicated. I'd also put a note about what the result is, like:

    x // result: [ 1, 8, 3, 4, 5 ]